book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] We want to allow a person to create a rental given a date and a person object. That is the purpose of this method. Kindly refactor the method:

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```